prog/mutation.go

```diff
@@ -80,10 +80,16 @@ func (p *Prog) Mutate(rs rand.Source, ncalls int, ct *ChoiceTable) {
 							default:
 								panic(fmt.Sprintf("bad arg kind for BufferType: %v", arg.Kind))
 							}
-							arg.Data = mutateData(r, data)
+							min_len := int(0)
```

Comment:
Go code does not use c++_naing_style. Please use camelCase for all identifiers.
s/min_len/minLen/